src/status_im2/contexts/wallet/home/view.cljs

```diff
@@ -27,56 +27,78 @@
       :on-press            #(rf/dispatch [:navigate-to :wallet-address-watch])
       :add-divider?        true}]]])
 
-(def account-cards
-  [{:name                "Account 1"
-    :balance             "€0.00"
-    :percentage-value    "€0.00"
-    :customization-color :blue
-    :type                :empty
-    :emoji               "🍑"
-    :on-press            #(rf/dispatch [:navigate-to :wallet-accounts])}
-   {:customization-color :blue
-    :on-press            #(rf/dispatch
-                           [:show-bottom-sheet {:content new-account}])
-    :type                :add-account}])
+(def add-account-placeholder
+  {:customization-color :blue
+   :on-press            #(rf/dispatch [:show-bottom-sheet {:content new-account}])
+   :type                :add-account})
 
 (def tabs-data
   [{:id :assets :label (i18n/label :t/assets) :accessibility-label :assets-tab}
    {:id :collectibles :label (i18n/label :t/collectibles) :accessibility-label :collectibles-tab}
    {:id :activity :label (i18n/label :t/activity) :accessibility-label :activity-tab}])
 
+(defn get-balance
+  [balances address]
+  (.toFixed (->> balances
+                 (filter #(= (:address %) address))
+                 first
+                 :balance)
+            2))
+
+(defn account-cards
+  [accounts loading? balances]
+  (let [refactored-accounts (mapv (fn [account]
+                                    (assoc account
+                                           :type                :empty
+                                           :customization-color :blue
+                                           :on-press            #(rf/dispatch [:navigate-to
+                                                                               :wallet-accounts])
+                                           :loading?            loading?
+                                           :balance             (str "$"
+                                                                     (get-balance balances
+                                                                                  (:address account)))))
+                                  accounts)]
+    (conj refactored-accounts add-account-placeholder)))
+
+(defn- view-internal
+  [accounts]
+  (let [top          (safe-area/get-top)
+        selected-tab (reagent/atom (:id (first tabs-data)))
```

Comment:
This was already implemented and used by another developer. This ticket is only for connecting data to the back end. We can create another ticket to fix the issues

Reply:
This is a recurring problem that keeps getting merged, I wish we could reliably detect this automatically, but we'd get some false positives I think.

Reply:
We can write a clj-kondo custom hook to detect this, But since it might be fragile we might only use it as a warning
Maybe something like:

1. is clause a defn?
2. does it contain hiccup?
3. if it's defining an atom then throw a warning.

Reply:
this should be resolved before merge

Reply:
its simpler to just move this into subscription